util/caller/resolver_test.go

```diff
@@ -58,3 +59,13 @@ func TestDefaultCallResolver(t *testing.T) {
 		}
 	}
 }
+
+func BenchmarkSimpleCaller(b *testing.B) {
+	for i := 0; i < b.N; i++ {
+		file, line, _ := Lookup(1)
+		s := fmt.Sprintf("%s:%d", file, line)
```

Comment:
Yeah, I was, but also trying to replicate what stopper does -- adding one inside, one outside makes it clearer where the overhead comes from.